xml/operating_system.xml

```diff
@@ -89,6 +90,34 @@
     <param pos="0" name="os.family" value="Windows"/>
     <param pos="1" name="os.product"/>
   </fingerprint>
+  <fingerprint pattern="^(?i:(?:Microsoft )?Windows(?:\sNT)? 6.1)$">
+    <description>Windows version 6.1 (Windows 7 or Windows Server 2008)</description>
+    <example>Windows 6.1</example>
+    <param pos="0" name="os.vendor" value="Microsoft"/>
+    <param pos="0" name="os.family" value="Windows"/>
+    <param pos="0" name="os.product" value="Windows 7 or Windows Server 2008"/>
```

Comment:
This should be Server 2008 R2 I believe, as 2008 is actually paired with Vista (6.0)